tools/src/main/java/org/apache/kafka/tools/streams/StreamsGroupCommand.java

```diff
@@ -270,7 +285,7 @@ private void printOffsets(StreamsGroupDescription description, boolean verbose)
                     String fmt =  "%" + (-groupLen) + "s %" + (-maxTopicLen) + "s %-10s %-15s %s\n";
                     System.out.printf(fmt, "GROUP", "TOPIC", "PARTITION", "LEADER-EPOCH", "OFFSET-LAG");
                     for (Map.Entry<TopicPartition, Long> offset : offsets.entrySet()) {
-                        System.out.printf(fmt, description.groupId(), offset.getKey().topic(), offset.getKey().partition(), "", offset.getValue());
+                        System.out.printf(fmt, description.groupId(), offset.getKey().topic(), offset.getKey().partition(), description.groupEpoch(), offset.getValue());
```

Comment:
is it (`description.groupEpoch()`), the `LEADER-EPOCH`?

Reply:
No, that's the group epoch.

I wonder if you could take a look at `ConsumerGroupCommand` and check how we fetch the leader epoch there. We need to fetch the committed offsets from the consumer offset topic. For now, I believe we have to call `adminClient.listConsumerGroupOffsets` - unfortunately it's named this way, but it will work for streams groups as well.

We need a ticket to introduce copies of `listConsumerGroupOffsets` etc. pp. that are named `listStreamGroupOffsets` groups (and do the same) - they use the same RPC under the hood.

I would personally just use the same columns as in the consumergroup tool - `"CURRENT-OFFSET", "LOG-END-OFFSET", "LAG",` .

Reply:
>I would personally just use the same columns as in the consumergroup tool - "CURRENT-OFFSET", "LOG-END-OFFSET", "LAG", .

You mean no epoch any more?